easybuild/easyblocks/i/icc.py

```diff
@@ -80,23 +91,24 @@ def sanity_check_step(self):
         """Custom sanity check paths for icc."""
 
         binprefix = "bin/intel64"
-        libprefix = "lib/intel64/lib"
+        libprefix = "lib/intel64"
         if LooseVersion(self.version) >= LooseVersion("2011"):
             if LooseVersion(self.version) <= LooseVersion("2011.3.174"):
                 binprefix = "bin"
             elif LooseVersion(self.version) >= LooseVersion("2013_sp1"):
                 binprefix = "bin"
-                libprefix = "lib/intel64/lib"
+                if LooseVersion(self.version) >= LooseVersion("2016"):
+                    libprefix = "lib/intel64_lin"
```

Comment:
if `intel64` is still available (e.g. as a symlink), stick to that, because they *will* change this again later (but will have to keep `intel64` around)